docs/src/pages/[platform]/connected-components/authenticator/customization/override-function-calls/example.react.mdx

```diff
@@ -0,0 +1,88 @@
+import { Tabs, Alert } from '@aws-amplify/ui-react';
+import { CURRENT_MAJOR_VERSIONS, PREV_MAJOR_VERSIONS  } from '@/data/frameworks';
+
+export const latestVersion = CURRENT_MAJOR_VERSIONS['react'];
+export const prevVersion = PREV_MAJOR_VERSIONS['react'];
+
+  <Tabs.Container defaultValue={latestVersion}>
+    <Tabs.List>
+      <Tabs.Item value={latestVersion}>ui-react v{latestVersion} (latest)</Tabs.Item>
+      <Tabs.Item value={prevVersion}>ui-react v{prevVersion}</Tabs.Item>
+    </Tabs.List>
+    <Tabs.Panel value={latestVersion}>
+      In version {latestVersion}, `Auth` function calls are imported directly as shown below. 
+      ```jsx{2}
+import { Amplify } from 'aws-amplify';
```

Comment:
Don't we have an e2e example we could link to here instead of writing example code? Believe there should be examples for every framework in the _examples_ directory

Reply:
We do have examples that use overrides but they aren't specifically focused on it and include unrelated stuff which I thought could be confusing for people.